internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -62,6 +64,44 @@ message GetOperationsResponse {
   repeated Operation operations = 1;
 }
 
+// TraceQueryParameters contains query paramters to find traces. For a detailed
+// definition of each field in this message, refer to `TraceQueryParameters` in
+// `jaeger.api_v3`.
+message TraceQueryParameters {
+  string service_name = 1;
+  string operation_name = 2;
+  map<string, string> attributes = 3;
+  google.protobuf.Timestamp start_time_min = 4 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Timestamp start_time_max = 5 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Duration duration_min = 6 [
+    (gogoproto.stdduration) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Duration duration_max = 7 [
+    (gogoproto.stdduration) = true,
+    (gogoproto.nullable) = false
+  ];
+  // max_traces is the maximum number of traces to return. If the number of traces
+  // matching the query is greater than this number, only the first max_traces
+  // traces will be returned.
+  int32 max_traces = 8;
```

Comment:
@yurishkuro I think this field is called `search_depth` in `api_v3`. Do we want the same wording here?

Reply:
yes